components/taxReceiptDocument.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { Document, Page, Text, View, StyleSheet, Image, Font } from '@react-pdf/renderer'
+import { formatUnixDateAsMMDDYYYY } from '../utils/date.utils'
+import { formatCurrency, formatFullName } from '../utils/formatting'
+
+const styles = StyleSheet.create({
+  image: {
+    width: 150,
+    margin: '0 auto',
+    height: 75
+  },
+  page: {
+    fontFamily: 'Lato',
+    color: '#212121',
+    fontSize: 14,
+    padding: 50
+  },
+  header: {
+    textAlign: 'center',
+    margin: '50 0 20 0'
+  },
+  section: {
+    margin: '20 0'
+  },
+  transactions: {
+    width: '75%'
+  },
+  transactionRow: {
+    display: 'flex',
+    flexDirection: 'row'
+  },
+  transactionRowItem: {
+    margin: '3 0',
+    width: 100
+  },
+  bold: {
+    fontFamily: 'Lato Bold'
+  }
+})
+
+Font.register({
+  family: 'Lato',
+  src: '/fonts/lato/Lato-Regular.ttf'
```

Comment:
did you see https://github.com/diegomura/react-pdf/issues/680 ? seems like maybe you have to import it in JS and then reference it here. if it doesn't work -- does the receipt look horrible without the custom fonts ?